cromwell.example.backends/cromwell.examples.conf

```diff
@@ -548,6 +548,9 @@ services {
     #     # Which bucket to use for storing or retrieving metadata JSON
     #     bucket = "carbonite-test-bucket"
     #
+    #     # Limit on bytes we attempt to read from the bucket when retrieving a JSON
```

Comment:
TOL There were some interesting observations in chat yesterday that this figure refers to uncompressed and not compressed size, not sure if that sort of detail is worth adding here.